### PR TITLE
#21 fixed

### PR DESCRIPTION
**Pull request by ashish0910**
add message string on delete issue #21
tested on chrome,Firefox
screenshot 2019-02-19 at 4 53 46 pm

### CODE REVIEW

**Comment by ashish0910** [comment]
#19 also fixed

**Comment by ashish0910** [comment]
issue #20 fixed. Please review @llaske @tarunsinghal92 .

**Comment by llaske** [comment]
Thanks for your work. BTW you should do a PR different for each bug. Please split your PR into 3.

**Comment by ashish0910** [comment]
Done @llaske

File: dashboard/views/classrooms.ejs [hunk 1]
```diff
@@ -73,7 +73,7 @@
 										        <a title="View Students" style="pointer-events: none;"><span class="student-icon" style="opacity: 0.4;"></span></a>
                                             <% } %>
                                             <a title="Edit Classroom" href="classrooms/edit/<%= data.classrooms[i]._id %>"><i class="material-icons text-muted">edit</i></a>
-                                            <a title="Delete Classroom" href="classrooms/delete/<%= data.classrooms[i]._id %>" onclick="return confirm(document.webL10n.get('DoYouWantDeleteClassroom',{classroom:'<%= data.classrooms[i].name %>'}))"><i class="material-icons text-muted">delete_forever</i></a>
+                                            <a title="Delete Classroom" href="classrooms/delete/<%= data.classrooms[i]._id %>" onclick="return confirm('Do you want to delete classroom: <%= data.classrooms[i].name %> ?')"><i class="material-icons text-muted">delete_forever</i></a>
```
Comment (tarunsinghal92) [inline dashboard/views/classrooms.ejs]: @ashish0910 

why did the localization was removed? Adding hardcoded English text looks weird to me.

Also, there are 6 commits and only +1/-1 line change. why is it so?

**Comment by ashish0910** [comment]
@tarunsinghal92 there were 6 commits because earlier 3 issues were fixed in this PR but then @llaske asked to create different pr for each issue so I had to delete those changes from this branch and created 2 more pr.

File: dashboard/views/classrooms.ejs [hunk 2]
```diff
@@ -73,7 +73,7 @@
 										        <a title="View Students" style="pointer-events: none;"><span class="student-icon" style="opacity: 0.4;"></span></a>
                                             <% } %>
                                             <a title="Edit Classroom" href="classrooms/edit/<%= data.classrooms[i]._id %>"><i class="material-icons text-muted">edit</i></a>
-                                            <a title="Delete Classroom" href="classrooms/delete/<%= data.classrooms[i]._id %>" onclick="return confirm(document.webL10n.get('DoYouWantDeleteClassroom',{classroom:'<%= data.classrooms[i].name %>'}))"><i class="material-icons text-muted">delete_forever</i></a>
+                                            <a title="Delete Classroom" href="classrooms/delete/<%= data.classrooms[i]._id %>" onclick="return confirm(document.webL10n.get('DoYouWantDeleteClassroom {classroom:<%= data.classrooms[i].name %>}'))"><i class="material-icons text-muted">delete_forever</i></a>
```
Comment (tarunsinghal92) [inline dashboard/views/classrooms.ejs]: 💃

**Pull request merged**